_includes/components/new-header.html

```diff
@@ -0,0 +1,54 @@
+<div class="usa-overlay"></div>
+<header class="usa-header usa-header--extended"><div class="usa-navbar">
+    <div class="usa-logo" id="extended-logo">
+        <em class="usa-logo__text"><a href="/" title="Home" aria-label="Home">
+            {{ header.title | default: site.title }} <sup>ALPHA</sup></a></em>
+    </div>
+    <button class="usa-menu-btn">Menu</button>
+</div>
+    <nav aria-label="Primary navigation" class="usa-nav">
+        <div class="usa-nav__inner"><button class="usa-nav__close"><img src="/assets/img/usa-icons/close.svg" role="img" alt="close"></button>
+
+
+
+            <ul class="usa-nav__primary usa-accordion"><li class="usa-nav__primary-item">
+                <button class="usa-accordion__button usa-nav__link {% if page.url contains '/about/' %} usa-current{% endif %}" aria-expanded="false" aria-controls="extended-nav-section-one"><span>About</span></button>
+
+                {{ page.active }}
+                <ul id="extended-nav-section-one" class="usa-nav__submenu"><li class="usa-nav__submenu-item">
+                    <a href="/about/" class="{% if page.url == '/about/' %} usa-current{% endif %}">Who we are</a>
+                </li><li class="usa-nav__submenu-item">
+                    <a href="about/people/" class="{% if page.url == '/about/people' %} usa-current{% endif %}">Meet us</a>
```

Comment:
@jaube05 when you are on the 'About' page and then use the drop-down menu to click on 'Meet us' you get a 404. You can test it at this URL: https://accessibility-qa.civicactions.com/about/.